src/stdafx.h

```diff
@@ -348,20 +348,8 @@ typedef unsigned char byte;
 #	define PERSONAL_DIR ""
 #endif
 
-/* Compile time assertions. Prefer c++0x static_assert().
- * Older compilers cannot evaluate some expressions at compile time,
- * typically when templates are involved, try assert_tcompile() in those cases. */
-#if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1600)
-#	define assert_compile(expr) static_assert(expr, #expr )
-#	define assert_tcompile(expr) assert_compile(expr)
-#elif defined(__OS2__)
-	/* Disabled for OS/2 */
-#	define assert_compile(expr)
-#	define assert_tcompile(expr) assert_compile(expr)
-#else
-#	define assert_compile(expr) typedef int __ct_assert__[1 - 2 * !(expr)]
-#	define assert_tcompile(expr) assert(expr)
-#endif
+#define assert_compile(expr) static_assert(expr, #expr)
+#define assert_tcompile(expr) assert_compile(expr)
```

Comment:
There are only 5 usages of `assert_tcompile` .. shouldn't we just replace them with `assert_compile` now?